plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMinioOrcConnectorTest.java

```diff
@@ -170,6 +171,12 @@ public void testDropAmbiguousRowFieldCaseSensitivity()
                 .hasStackTraceContaining("Multiple entries with same key");
     }
 
+    @Override
+    public void testCorruptedTableLocation()
+    {
+        throw new SkipException("Skipping test, This test override will be removed in next commit");
```

Comment:
nit It's not clear from the exception why the skipping is being done - even if it is temporary.